### PR TITLE
Restore `ci-link` feature

### LINKED ISSUE #4260: `ci-link` is broken
Github now lazy loads the commit checks 😢 

Screen Shot 2

### PR DESCRIPTION
**Pull request by yakov116**
Fixes #4260
## Test URLs
Here and I tested on private

## Screenshot

image

### CODE REVIEW

**Comment by fregante** [comment]
Needs more work just for the alignment? You can target it via CSS if that's troubling you.

**Comment by yakov116** [comment]
> Needs more work just for the alignment? 

Yes

File: source/features/ci-link.tsx [hunk 1]
```diff
@@ -21,13 +21,15 @@ async function init(): Promise<false | void> {
 		return false;
 	}
 
-	icon.classList.add('rgh-ci-link');
+	icon.querySelector('.octicon-check')?.classList.add('v-align-middle');
```
Comment (yakov116) [inline source/features/ci-link.tsx:24]:
```suggestion
```

**Comment by fregante** [comment]
gif

Ugly :(

Other than that, the look seems to match the last screenshot exactly (July 2020) https://github.com/sindresorhus/refined-github/pull/3362#discussion_r456570554

Can you target the loading animation via CSS and hide it? Also I think we can drop the fade-in code everywhere (callCount and that stuff)

**Comment by kidonng** [comment]
I guess the selector needs to be more specific:

image

*Yo! My fork has been verified by Refined GitHub*

**Comment by yakov116** [comment]
LOL

**Comment by yakov116** [comment]
We should add `[data-url$="commits/checks-statuses-rollups"]`

**Comment by gustavkj** [comment]
I think this might have broken `ci-links` for me on Github Enterprise. We're currently running GHE version `2.22.4` and I'm using the latest Github Refined (`21.4.23`).

**Comment by yakov116** [comment]
> I think this might have broken `ci-links` for me on Github Enterprise. We're currently running GHE version `2.22.4` and I'm using the latest Refined Github (`21.4.23`).

What part broke? Picture? Console error?

**Comment by gustavkj** [comment]
> What part broke? Picture? Console error?


The CI link does not show up anymore. The reason is because the Github Enterprise instance has not been updated, so the updated selector does not work yet. 

In the screenshot below I used the two current selectors first and the removed selector on the `commits` page. As you can see, the current ones do not work, the one that was replaced does:


Screenshot 2021-04-28 at 14 04 11

**Comment by yakov116** [comment]
ohh I see the issue you are not on the old layout but you still have the non lazy loaded version

**Comment by fregante** [comment]
We can add the old selector to the array. No animation needed

**Comment by gustavkj** [comment]
> We can add the old selector to the array. No animation needed

That would be much appreciated! 🙂

**Pull request merged**